### PR TITLE
Staging - Cleaned UI and updated features

### LINKED ISSUE #3: Dashboard UI
- [ ] Auth page before landing (update the stock looking login/reg page)
- [x] Markdown support added (`React-markdown` package)
- Landing page
   - [x] UI updated + refactored folders + remove extras
   - [x] Search box and mock results
   - [ ] To-do list area

### PR DESCRIPTION
**Pull request by sauravhiremath**
- [x] Integrated the GitHub notifications view by @darshkpatel 
- [x] Extra widgets cleaned and folder restructured
- [x] The theme for primary-color updated to match the teal color
- [x] Updated metadata for index.html
- [x] Added logo
- [x] add .env to .gitignore -_-
- [x] Merged server code

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 PORT=3000
 CHOKIDAR_USEPOLLING=true
+REACT_APP_GITHUB_TOKEN=37fe0fb6ffd68943576cdb020d456db2ebb2176e
```
Comment (sauravhiremath) [inline .env:3]: No need to panic! I've removed the oauth token no longer valid.

Also, why isn't `.env` in .gitignore -_-

Comment (darshkpatel) [reply .env:3]: well, this repo wasn't made using create-react-app it's missing a lot of things

**Review by darshkpatel — COMMENTED** [review]
`src/lib/data/allUsers.json → src/views/notifications/github/data/allUsers.json`
`src/lib/githubAPI.js → src/views/notifications/github/githubAPI.js`

Why do these files belong to the notifications folder? or were they accidentally moved while renaming stuff

**Comment by sauravhiremath** [comment]
> `src/lib/data/allUsers.json → src/views/notifications/github/data/allUsers.json`
> `src/lib/githubAPI.js → src/views/notifications/github/githubAPI.js`
> 
> Why do these files belong to the notifications folder? or were they accidentally moved while renaming stuff

Yeah I accidentally moved them all into notifications. Will keep the githubAPI.js and its data in scripts folder

**Pull request merged**